### DISCUSSION TITLE
Welcome to Tasmota Discussions!

### DISCUSSION THREAD

**Discussion by arendst** [arendst/Tasmota#10124] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arendst** [comment]
Oh oh.

**Reply by ziotprojects** [reply]
How can I change all the English messages or texts in Tasmota.bin to support Thai language. I saw your Tasmota.bin has many languages support.

**Reply by Frogmore42** [reply]
Probably best to ask this as a new question

**Reply by ascillato2** [reply]
https://github.com/arendst/Tasmota/discussions/10128

**Comment by meingraham** [comment]
👍

**Comment by buteomont** [comment]
I hope this is the correct forum to ask this question.
I have three Sonoff Basics running Tasmota 8.3.1 at remote locations, communicating via MQTT to a central broker.  My goal is to get informed if the power at those locations goes off.  The most straightforward way of detecting this is to monitor the LWT of these Sonoffs, and send a notification when one of them sends the LWT "Offline" status.
This seems like a good way to accomplish this, but I occasionally get a rash of false reports.  I'm no expert on MQTT, but the best I can tell is that if the broker doesn't get a ping from the device in a timely manner, it will publish the LWT message.  I believe this timeout value is set by the client, not by the broker, so Tasmota must be either setting too short of a timeout value in the broker, or it is not sending pings frequently enough to meet this timeout value.
Am I way off base here, or is there a bug in Tasmota?

**Reply by ascillato2** [reply]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Remember to read the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md). Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Community](https://groups.google.com/d/forum/sonoffusers) for forum.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by Frogmore42** [comment]
Might want to clarify the purpose of discussions and the use of them.  Here is how I see it.

1. Chat is great if you want real time help with a problem that you can't figure out.
1. Chat is very poor for searching for help when what you have is probably a simple "I've never done this before and it isn't working like the information says it should.
1. Discussions allow for proper threading, but only if people start a new discussion (instead of just reusing this one.
1. Discussions could be a replacement for Community.  Community is yet another place/username/password to use, so I rarely use it
1. Discussions are a great place to have discussions (that are not issues) that don't need to be real time (or real time is not possible due to significant time differences).

Right now there is not a lot of activity on Discussions.  I believe this is likely because they are new (both here and to GitHub), but as people learn they exist I suspect they will get more activity.

**Comment by eddee54455** [comment]
Hi Guys, I am an absolute noob with Tasmota relatively speaking... 

I am running quite a few Tasmota flashed Sonoffs doing weird and useful things around my plot.... One thing I would dearly like to do is use my small "gate remote" transmitters that every member of the family has, to do some outside light switching... They are unfortunately of the "Keeloq" variety and the standard RF protocols on RFrecv configuration does not seem to support it.... 

I am using a SRX882 and a STX882 to play around with... Both are interfaced into Sonoff Basic R2 and R3 respectively... They are working, able to send data to each other via console...

When I transmit from a "Gate Remote" transmitter, the SRX does receive the data and push it out the data line, but the Tasmota side of things does not register the data on the console... (I have it hooked up to my old scope and can see the square wave binary pattern being rec'd)

Any idea how I could get Tasmota to read this data? I am not needing the entire code word, which includes the encrypted data, simply the Serial Number and Button Status... The chipset used in the "Gate Remote" is a HCS301... I have included the data sheet for good measure, the section explaining the data layout is Figure 4-2 ....

Any assistance would be AWESOME!!

Regds
Ed

Oh, and yes, many people have said it before, but it needs to be said again: "Mr Theo Arends, you are a fantastic and generous programmer to put your hard earned time and knowledge onto the web for all of us to use 
[HCS301 Rempte Button Fob Chipset.pdf](https://github.com/arendst/Tasmota/files/5712073/HCS301.Rempte.Button.Fob.Chipset.pdf)
- THANK YOU!!"

**Reply by Frogmore42** [reply]
The point of discussions is to keep them focused.  Please start a NEW one for your idea/request.
Go here: https://github.com/arendst/Tasmota/discussions
Click on the "New discussion" button (the green one).  "Ideas" would the appropriate category for your idea.  This discussion is about Discussions.

**Comment by Frogmore42** [comment]
If you have a comment about the use of Discussions for Tasmota, this is the place to make it.  If you actually want to have a discussion, please create a new one like this:

> Go here: https://github.com/arendst/Tasmota/discussions
> Click on the "New discussion" button (the green one). Pick the appropriate category for your discussion. This discussion is about Discussions.

I think I like the new Discussions feature.  It is not getting much use yet, but I believe it will as more people discover it.

**Comment by NEoKhajitt** [comment]


**Reply by Frogmore42** [reply]
Just curious, did you miss this in the item above?

>If you actually want to have a discussion, please create a new one like this:

> Go here: https://github.com/arendst/Tasmota/discussions

> Click on the "New discussion" button (the green one). Pick the appropriate category for your discussion. This discussion is about Discussions.

**Comment by elik745i** [comment]
This is my weather station project, where I've used tasmota, specially compiled firmware. I had to write my first driver for digital compass. Feel free to use translator, maybe in the future I'll post it in eng, but need some time...
Thanks to tasmota support and community. This was really good experience and quite challenging project.
https://3dtoday.ru/blogs/elik745i/moya-pogodnaya-stanciya

**Reply by sfromis** [reply]
Better post such as a new topic in "Show and tell". People are not going to find it when it is buried on another thread.

**Reply by elik745i** [reply]
DONE!
